ldelf/ta_elf_rel.c

```diff
@@ -185,6 +202,12 @@ static void e32_relocate(struct ta_elf *elf, unsigned int rel_sidx)
 
 		str_tab_idx = shdr[sym_tab_idx].sh_link;
 		if (str_tab_idx) {
+			if (str_tab_idx >= elf->e_shnum)
+				err(TEE_ERROR_GENERIC,
+				    "Strtab index out of range");
```

Comment:
`STRTAB` (all uppercase) perhaps? That's the way `readelf` prints it.

Reply:
Makes sense, but there's quite a few other places which should be updated too then.
OK if I take that in a follow up PR so I can update all the other places too?

Reply:
Fine by me